### PR TITLE
Automate deployment

### PR DESCRIPTION
**Pull request by ChrisTerBeke**
Use CuraPackageDeployer to automatically build and deploy this plugin. Each branch will automatically upload the plugin sources to the Ultimaker packages API for building. The master branch will trigger a new release (or update an existing one with the correct version number) and submit the release for review.

### CODE REVIEW

**Comment by ChrisTerBeke** [comment]
Improvements I want to make later (to the library) are:

- Using the API to check the build status and fail the CI if it cannot build your plugin
- Do some more local validation of required files before attempting to upload the sources
- Add tests

File: deploy.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    deployer = CuraPackageDeployer(config)
+    deployer.loadPluginSources()
+    deployer.buildPlugin()
+    
+    if os.getenv("DEPLOY") == "True":
+        deployer.deploy()
+        time.sleep(3)  # Give the API some time to build the package.
+        deployer.requestReview()
+        
+    return 0
```
Comment (DanielSchiavini) [inline deploy.py]: return code is always zero, no need to set it then

File: deploy.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+load_dotenv()
+
+
+class CuraDriveConfig(Config):
+    package_id = "CuraDrive"
+    package_sources_dir = "./CuraDrive"
+    tags = ["backups", "cloud", "restore", "configuration", "settings", "sync"]
+    website = "https://ultimaker.com"
+    release_notes = "Switched to the new Ultimaker Account functionality baked into Cura."
+    access_token = os.getenv("ACCESS_TOKEN", None)
```
Comment (DanielSchiavini) [inline deploy.py]: What happens if the access token is empty?

Comment (ChrisTerBeke) [reply deploy.py]: The 1st API call will fail, so the CI will fail. The library is not 100% robust yet (that's why it's on my own account still) and definitely needs more validation, but then again it's an automation tool used by a small number of people.

File: deploy.py (hunk 1 above)
Comment (ChrisTerBeke) [reply deploy.py]: Fixed.

**Pull request merged**